frontend/src/components/Candidate-Decider/CandidateDecider.tsx

```diff
@@ -129,9 +129,9 @@ const CandidateDecider: React.FC<CandidateDeciderProps> = ({ uuid }) => {
           <Button
             className="ui blue button"
             disabled={currentComment === getComment() && currentRating === getRating()}
-            onClick={() => {
-              handleRatingAndCommentChange(currentCandidate, currentRating, currentComment);
-            }}
+            onClick={() =>
+              handleRatingAndCommentChange(currentCandidate, currentRating, currentComment)
+            }
```

Comment:
i'd get rid of this for git history reasons

Reply:
agreed